internal/provider/provider.go

```diff
@@ -67,6 +70,17 @@ func configure(p *schema.Provider) schema.ConfigureContextFunc {
 		if err != nil {
 			return nil, diag.FromErr(err)
 		}
+
+		// check harvester version from settings
+		serverVersion, err := c.HarvesterClient.HarvesterhciV1beta1().Settings().Get(context.Background(), "server-version", metav1.GetOptions{})
+		if err != nil {
+			return nil, diag.FromErr(err)
+		}
+		// harvester version v1.0-head, v1.0.2, v1.0.3 is not supported
+		if strings.HasPrefix(serverVersion.Value, "v1.0") {
+			return nil, diag.FromErr(fmt.Errorf("current harvester server version is %s, only support v1.1+", serverVersion.Value))
```

Comment:
\s\harvester\Harvester
\s\only support v1.1+\the minimum supported version is v1.1.0.

Reply:
updated